example/client.js

```diff
@@ -4,12 +4,14 @@ var http2 = require('..');
 
 // Setting the global logger (optional)
 http2.globalAgent = new http2.Agent({
+  // Avoid verification errors for self-signed certs.
```

Comment:
I would rather have the key and ca options gated on the request going to localhost - that way our example client (which could be used as a simple curl or wget-like download client) doesn't allow that key to be used by just any old host on the internet. If we do that, though, then this would be a strict improvement over the current use of NODE_TLS_REJECT_UNAUTHORIZED.